suite/suite_test.go

```diff
@@ -259,10 +259,12 @@ func (suite *SuiteTester) TestSubtest() {
 
 func (suite *SuiteTester) TearDownSubTest() {
 	suite.TearDownSubTestRunCount++
+	suite.Contains(suite.T().Name(), "subtest", "We should get the *testing.T for the test that is to be torn down")
```

Comment:
Updating the test suite attributes here and performing assertions alongside the others in the main test body will be more consistent with the current testing setup:
https://github.com/stretchr/testify/blob/ac5cd69d4baa639ccd1a95c52d627827e4ed242d/suite/suite_test.go#L154-L172

See this for reference:
https://github.com/stretchr/testify/pull/1393/files#diff-ca4ffdb7050ad570420ac5682fefad8481384709c2884051cd57071283ce9ca4